api/src/test/java/org/openmrs/api/ProgramWorkflowServiceTest.java

```diff
@@ -56,6 +57,8 @@
 	protected EncounterService encounterService = null;
 	
 	protected ConceptService cs = null;
+
+	protected ProgramWorkflowDAO dao = null;
```

Comment:
You should be testing ProgramWorkflowService and not DAO, so you will need to call the service and not the DAO

Reply:
@dkayiwa I will go through it again. Thanks.